### PR TITLE
on OS X use staticfloat/juliadeps/imagemagick@6

### PR DESCRIPTION
**Pull request by bjarthur**
this PR fixes the mess that homebrew created a few days ago when they updated their formulae to only support libImageMagick v7.0.4.  we now directly install the v6 bottle kindly provided by @staticfloat in Homebrew.jl.

while at it i also took the liberty to update the README a bit, specifically mentioning that ImageMagick.jl only supports libimagemagick v6.

### CODE REVIEW

**Comment by bjarthur** [comment]
closes https://github.com/JuliaIO/ImageMagick.jl/issues/57

see https://github.com/staticfloat/homebrew-juliadeps/issues/114

File: README.md [hunk 1]
```diff
@@ -33,10 +36,9 @@ It's worth pointing out that packages such as Images load FileIO.
 
 ## OSX
 
-ImageMagick.jl will by default not use the system installed ImageMagic in
-`/usr/local`, but rather install it's own copy via Homebrew.jl, because the
-former results in problems like `ERROR: no encode delegate for this image format
-'MIFF'`.
+ImageMagick.jl will use the system-wide libMagicWand in `/usr/local/lib` if
+present (e.g. via a manual installation with homebrew).  If not it will install
+it's own copy via Homebrew.jl.
```
Comment (timholy) [inline README.md]: Delete apostrophe

**Comment by timholy** [comment]
LGTM. Thanks, Ben.

**Comment by ilovezfs** [comment]
@bjarthur @timholy Just a heads up that we've now merged ImageMagick 6 as "imagemagick@6"

**Pull request merged**